src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/IncentiveLevelService.kt

```diff
@@ -97,7 +108,11 @@ class IncentiveLevelService(
       )
     }
 
-    return incentiveLevelRepository.saveAll(incentiveLevelsWithNewSequences).toListOfDTO()
+    return incentiveLevelRepository.saveAll(incentiveLevelsWithNewSequences)
+      .toListOfDTO()
+      .also {
+        auditService.sendMessage(INCENTIVE_LEVELS_REORDERED, "*", it.map(IncentiveLevelDTO::code))
```

Comment:
I'm not sure how the ID is used on the other side of the fence. I'm guessing it's not, just stored for reference/displayed somewhere.

However, even if that's so, I'd probably prefer something more specific here.

I know this is an operation which didn't involve a single entity but what do you think about having the list of incentive codes here instead? e.g. `"BAS, STD, ENH, EN2, EN3"` - it's an idea.